model/model.mustache

```diff
@@ -75,28 +75,22 @@ module.exports = {
       return Promise.reject(new httpError('BAD_REQUEST', 'No rev provided'));
     }
 
-    return r.table('{{pluralName}}').get({{name}}.id).run()
-    .then(current => {
-      const updated = Object.assign({}, current, {{name}});
-      const validate = schema.getValidator('{{name}}');
-      const valid = validate(updated);
-      if (!valid) throw new httpError('UNPROCESSABLE_ENTITY');
+    const valid = validate({{name}});
+    if (!valid) throw new httpError('UNPROCESSABLE_ENTITY');
```

Comment:
This needs to return a rejected promise since it's now outside the promise chain. In hosportal I've just wrapped the entire validation in a try catch (https://github.com/Prismatik/hosportal/blob/master/api/models/shift_template.js#L62), but the new Promise approach i mentioned is probably nicer.